samples/annotate/pom.xml

```diff
@@ -61,7 +61,12 @@
 		</dependency>
 		<dependency>
 			<groupId>com.sun.xml.bind</groupId>
-			<artifactId>jaxb-impl</artifactId>
+			<artifactId>jaxb-core</artifactId>
+		</dependency>
+		<dependency>
```

Comment:
Let's switch to slf4j completely.

On Fri, Feb 19, 2016 at 12:16 PM, Andrii Matsokha <notifications@github.com>
wrote:

> In samples/annotate/pom.xml
> <https://github.com/highsource/jaxb2-annotate-plugin/pull/23#discussion_r53449116>
> :
>
> > @@ -61,7 +61,12 @@
> >  		</dependency>
> >  		<dependency>
> >  			<groupId>com.sun.xml.bind</groupId>
> > -			<artifactId>jaxb-impl</artifactId>
> > +			<artifactId>jaxb-core</artifactId>
> > +		</dependency>
> > +		<dependency>
>
> We could use log4j-jcl adapter instead of commons-logging, would it be
> better?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/highsource/jaxb2-annotate-plugin/pull/23/files#r53449116>
> .
>